components/authentication/base/segment-bridge.yaml

```diff
@@ -0,0 +1,64 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Can you place it on a dedicated argocd application set? we are trying to reduce the manifests in the `authentication` component (you can do it in a different PR).